path_role.go

```diff
@@ -128,6 +129,9 @@ authenticate against this role`,
 				Type:        framework.TypeCommaStringSlice,
 				Description: `Comma-separated list of allowed values for redirect_uri`,
 			},
+			"verbose_oidc_logging": {
+				Type: framework.TypeBool,
```

Comment:
Since this will be perpetuated to the UI, might be nice here to add a description that includes a warning that it causes the logging of sensitive information. Not the kind of stuff you'd want picked up by filebeat and sent off to ELK, probably.

Reply:
Yeah. I'm going to add a note here, and probably a warning in the config write response if you're enabling this.